config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-periodic-dra.yaml

```diff
@@ -27,9 +27,9 @@ periodics:
       repo: kubernetes
       base_ref: master
       path_alias: k8s.io/kubernetes
-    - org: kubernetes
+    - org: nojnhuh
```

Comment:
Is this intended?

Reply:
Yes, we want to get runs with these changes ASAP before a PR upstream could probably be merged.